### PR TITLE
fix: should catch Throwable rather than Exception

### PR DESCRIPTION
**Pull request by spacekitcat**
In PHP 7, you should use Throwable if you want to catch all errors.

### CODE REVIEW

**Review by murkas — APPROVED** [review]
Thank you for the description added, I was not aware of this feature in PHP7. This seems like a good catch-all.

File: src/Branding/LogoVersionStrategy.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ protected function loadLogoConfig($forceRefresh = false): LogoConfig
                 return $json;
             });
             $this->logoConfig = LogoConfig::fromJSON($json);
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```
Comment (spacekitcat) [inline src/Branding/LogoVersionStrategy.php:119]: It is arguably bad practice to catch Throwable here, but the problem is that `fromJSON` appears to throw a TypeError when it fails to parse the contents of `$json`. We need to remove the unparsable payload from the cache, so I think this context is an exception.

Comment (ajroberts10) [reply src/Branding/LogoVersionStrategy.php:119]: We could always just add an extra catch block for `TypeError` if we didn't want to be so absolute although we probably just want to delete this key in the event of any error right?

Comment (ajroberts10) [reply src/Branding/LogoVersionStrategy.php:119]: Maybe having an additional catch block here means we could add some logging specific to this issue, possibly making it easier to identify in the logs?

**Pull request closed without merging**